# debian_alsaconf_idmode.patch by Thomas Hood
#
# Use Debian-specified GID and mode for ALSA devices.
Index: alsa-utils-1.0.18/alsaconf/alsaconf.in
===================================================================
--- alsa-utils-1.0.18.orig/alsaconf/alsaconf.in	2008-11-02 20:59:25.000000000 +0100
+++ alsa-utils-1.0.18/alsaconf/alsaconf.in	2008-11-02 20:59:25.000000000 +0100
@@ -110,8 +110,8 @@
 do_legacy_only=0
 use_modinfo_db=0
 alsa_uid=0
-alsa_gid=0
-alsa_mode=0666
+alsa_gid=29
+alsa_mode=0660
 legacy_probe_card=""
 LOGFILE=""
 TESTSOUND="@TESTSOUND@"
